e2e/config/config.go

```diff
@@ -237,6 +237,31 @@ func (c configTests) configGlobal(t *testing.T) {
 			exit:           0,
 			resultOp:       e2e.ExpectOutput(e2e.ExactMatch, "1"),
 		},
+		{
+			name:           "AllowUserNsNo",
+			argv:           []string{"--userns", c.env.ImagePath, "true"},
```

Comment:
In e2e tests by convention we don't use `--userns`. Instead we use the `e2e.UserNamespaceProfile` which will add the flag for the test.